docker/dev_env/Dockerfile

```
@@ -56,6 +58,7 @@ RUN mkdir /pipx \
     python3.12 pipx \
     docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin \
     unzip \
+    postgresql postgresql-server \
```

The PostgreSQL server package is not needed in the container. Additionally, the `postgresql-devel` package is required.

```suggestion
    postgresql postgresql-devel \
```